core/queryalgebra/evaluation/src/main/java/org/eclipse/rdf4j/query/algebra/evaluation/iterator/DescribeIteration.java

```diff
@@ -202,7 +202,7 @@ protected BindingSet getNextElement() throws QueryEvaluationException {
 		return null;
 	}
 
-	private CloseableIteration<BindingSet, QueryEvaluationException> createNextIteration(Value subject, Value object)
+	protected CloseableIteration<BindingSet, QueryEvaluationException> createNextIteration(Value subject, Value object)
```

Comment:
@jeenbroekstra are you OK with making this method and some fields `protected` to allow reusing this implementation logic? Otherwise I would need to duplicate kind of the same code. Or do you perhaps see an alternative way of implementation in the federation?

Reply:
@aschwarte10 I think this is fine for these purposes. Making things protected does not make them part of the public API as far as I'm concerned, and here there's a good reason to do it.